### DISCUSSION TITLE
Jans is blocking all the incoming traffic through ingress for the other services.

### DISCUSSION THREAD

**Discussion by sriniazhagar** [JanssenProject/jans#6502] [Q&A] [answered]
"We installed Jans in a K8 cluster. This cluster is a shared cluster. In this shared cluster we have different shared services like cache service, Dyna Trace, etc. They have their own namespace in this shared cluster. "
The Problem: After installing the Jans in this shared cluster, it works fine. But then all on a sudden (we can not predict when this will happen", it seems Jans is blocking all the incoming traffic through ingress for the other services.

Question 1: Is this a known behavior of Jans? Does it need a complete dedicated cluster of its own?
Question 2: As Jans internally uses Nginx Ingress, have you patched the vulnerability?

**Comment by misba7** [comment, marked as answer]
Hello.
Jans doesn't have to be in a dedicated cluster nor does it block other Ingress traffic.  
I even double-checked on my end and deployed another app beside Jans and ingress worked fine as you can see in the screenshot.

![Screenshot (907)](https://github.com/JanssenProject/jans/assets/49232835/b8e9e801-eec7-454b-821e-6590183191a9)

**Reply by sriniazhagar** [reply]
Thanks for the update and response, will check from end.
